backend/src/services/secret/secret-service.ts

```diff
@@ -951,7 +966,23 @@ export const secretServiceFactory = ({
     expandSecretReferences,
     recursive
   }: TGetSecretsRawDTO) => {
-    const botKey = await projectBotService.getBotKey(projectId);
+    const { botKey, shouldUseSecretV2Bridge } = await projectBotService.getBotKey(projectId);
+    if (shouldUseSecretV2Bridge) {
+      const { secrets, imports } = await secretV2BridgeService.getSecrets({
```

Comment:
I didn't use that name because this is not the actual engine. This is just a backward port for compatibility. When we develop the new secret endpoints with all changes needed that will be a secret engine. After the depreciation period these will be deleted.

Reply:
Update: makes sense, the bridge is just to make sup service but where only the secret data source changes. In the future, we will have the real secret engine v2 developed along with new end points